src/_content/posts/introducing-keynote-mario-munoz.md

```diff
@@ -0,0 +1,59 @@
+---
+author: DjangoCon US Communications Team
+category: General
+date: 2024-08-01 06:00:00
+cover:
+    url: /assets/img/blog/mario_munoz.jpg
+    alt: "Mario Munoz"
+layout: post
+title: "Announcing Our Keynotes: Mario Munoz"
+---
+
+### We're honored to announce our seecond keynote speaker for DjangoCon US 2024: Mario Munoz!
+
+We had the opportunity to sit down with Mario and ask her a few questions about his work, his upcoming talk, and what shehee's looking forward to at DjangoCon US 2024.
+
+## Can you tell us how you use Django
+
+>When preparing my first talk on "why I didn't choose Django," I thought I should at least build a Django app before getting into it... As a result, I built an app to hold my presentation slides (also using reveal.js), and ever since, I've continued using it for the past two years.
```

Comment:
```suggestion
>When preparing my first talk on ["Why I Didn't Start With Django"](https://2022.djangocon.us/talks/why-i-didn-t-start-with-django/), I thought I should at least build a Django app before getting into it... As a result, I built an app to hold my presentation slides (also using reveal.js), and ever since, I've continued using it for the past two years.
```